e2e/tests/security.spec.ts

```
@@ -0,0 +1,508 @@
+      ];
+      
+      for (const payload of xssPayloads) {
+        // Look for email/username input
+        const emailInput = page.locator('input[type="email"], input[name="email"], input[placeholder*="email" i]').first();
+        
+        if (await emailInput.isVisible()) {
+          await emailInput.fill(payload);
+          
+          // Check that script is not executed
+          const hasAlert = await page.evaluate(() => {
+            return window.alert !== window.alert; // Check if alert was overridden
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)